contracts/ForkableBridge.sol

```diff
@@ -185,7 +185,7 @@ contract ForkableBridge is
         uint256 amount,
         bool useFirstChild,
         bool useChildTokenAllowance
-    ) public onlyForkManger onlyAfterForking {
+    ) public onlyAfterForking {
```

Comment:
Since all tokens are no longer accessible in the old bridge contract and they need to be withdrawn in the new contract, I think its fine if everyone can push them to the new chain. But I will rework this in one of my next PRs anyway